### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by rpire**
## Changes made:

- Added media queries to reposition elements in the webpage.
- Re-adjusted the attributes of elements for desktop displays.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,147 +10,182 @@
...
+      <h1>Charlie Giorgania</h1>
+      <p>
+        Hello! I am a software developer! I can help you build a product, feature
+        or website. Taka a look of my works. If you like what you see and have a
+        project you need coded, don’t hesitate and contact me.
+      </p>
+      <a class="collab-link" href="#" rel="noopener noreferrer">
+        <div class="collab-button">Start Collaboration</div>
+      </a>
+      <img class="deco-0" src="./Media/image_dots.svg" alt=" ">
```
Comment (Shaher-11) [inline index.html:37]: Reduce the image size as the difference shown below
#### Original design
right

#### Your Project
wrong

File: index.html [hunk 2]
```diff
@@ -10,147 +10,182 @@
...
-  <section class="about">
-    <h2>About</h2>
-    <p>
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-      veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea
-      eiusmod
-    </p>
-    <div class="about-container">
+  <section id="about-wrapper" class="about">
```
Comment (Shaher-11) [inline index.html:172]: The space between the image and the text should be reduced 
### Original design
right

#### Your Project
wrong

File: index.html [hunk 3]
```diff
@@ -183,9 +218,10 @@ <h3>Skills</h3>
         </ul>
         <img src="./Media/ic_skills.svg" alt=" " class="about-img">
       </div>
+      <img class="deco-7" src="./Media/image_dots_with_teal.svg" alt=" ">
     </div>
   </section>
-  <section class="contact-me">
+  <section id="contact" class="contact-me">
```
Comment (Shaher-11) [inline index.html:224]: Please ignore this part of the design template and use only one input for the full name for all versions.
wrong

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @rpire,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [x] Check the comments below:point_down:

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

**Comment by rpire** [comment]
## Hello @Shaher-11

Thanks for reviewing my work, I will make sure to complete all changes requested before requesting another review. 🤓

Once again, thank you. Cheers and Happy coding! 🚀

File: index.html (hunk 1 above)
Comment (rpire) [reply index.html:37]: Changed applied successfully.

File: index.html (hunk 2 above)
Comment (rpire) [reply index.html:172]: Change applied successfully.

File: index.html (hunk 3 above)
Comment (rpire) [reply index.html:224]: Template 5 requires two inputs, name and email. Kept both inputs in the same row just like in the template but reduced the distance between them.

**Comment by rpire** [comment]
## Thank you @Shaher-11 

All requested changes have been implemented successfully and my portfolio looks much better now!

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi @rpire, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Comment by rpire** [comment]
## Thank you! 🙌

Greeting @arslanbisharat 

Thank you for reviewing my work, I will merge it right away. 🎉🎉

Cheers and Happy coding!  🚀

**Pull request merged**